sdk_contrib/typed_span/src/main/java/io/opentelemetry/sdk/contrib/typedspan/http/HttpClientSpanWrapper.java

```diff
@@ -0,0 +1,31 @@
+package io.opentelemetry.sdk.contrib.typedspan.http;
+
+import static io.opentelemetry.sdk.contrib.typedspan.http.HttpSpanWrapperBuilder.STATUS_CODE_KEY;
+import static io.opentelemetry.sdk.contrib.typedspan.http.HttpSpanWrapperBuilder.STATUS_TEXT_KEY;
+
+import io.opentelemetry.trace.Span;
+import io.opentelemetry.trace.Status;
+
+public class HttpClientSpanWrapper extends HttpSpanWrapper {
+  public HttpClientSpanWrapper(Span span) {
+    super(span);
+  }
+
+  public HttpClientSpanWrapper setStatusCode(HttpStatusCode status) {
+    this.getRawSpan().setAttribute(STATUS_CODE_KEY, status.getCode());
+    this.getRawSpan().setAttribute(STATUS_TEXT_KEY, status.getMsg());
```

Comment:
I think you should not set the status text if you know it's just the default that adds no information beyond the status code integer.

Reply:
The [spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/data-http.md#common-attributes) says that it should be the default [HTTP Status line](https://tools.ietf.org/html/rfc7230#section-3.1.2)

Reply:
Yes but it's optional. And in the case of a fixed string that is known from the status code, I'd say, leave it out.